proto/pulumi/language.proto

```diff
@@ -164,6 +164,8 @@ message GenerateProjectRequest {
     bool strict = 4;
     // The target of a codegen.LoaderServer to use for loading schemas.
     string loader_target = 5;
+    // local dependencies to use instead of using the package system
+    map<string, string> local_dependencies = 6;
```

Comment:
What exactly is in the map? Key-value pairs of what?

I'm not a big fan of maps at API boundaries where something more structured (e.g. `repeated LocalDependency`, `message LocalDependency { name, local_path, etc. }`) would be more informative, but I don't feel strongly about that here because this API is likely to change in the near future anyway.

Reply:
It's a "package name" -> "local path to artifact to use". 
I can extend the comment for now, but I'd rather keep this a map then a list of Dep{Name,Path} just to make it clear that the name can only show up once.